impl/src/main/java/dates/RWDateFormat.java

```diff
@@ -0,0 +1,32 @@
+package dates;
```

Comment:
We might want to add a comment here and in JaxRsModule that these are actually bug workarounds for the framework and should be removed once we fix that there.